java/code/src/com/suse/manager/webui/utils/salt/custom/DistUpgradeSlsResult.java

```diff
@@ -32,13 +28,13 @@ public class DistUpgradeSlsResult {
    @SerializedName("mgrcompat_|-spmigration_|-pkg.upgrade_|-module_run")
    private StateApplyResult<RetOpt<Map<String, Change<String>>>> spmigration;
 
-   @SerializedName("cmd_|-spmigration_|-cat /var/log/yum_sles_es_migration.log_|-run")
+   @SerializedName("cmd_|-spmigration_|-/usr/bin/cat /var/log/yum_sles_es_migration.log_|-run")
    private StateApplyResult<CmdResult> liberateYumEs;
 
-   @SerializedName("cmd_|-spmigration_|-cat /var/log/dnf_sles_es_migration.log_|-run")
+   @SerializedName("cmd_|-spmigration_|-/usr/bin/cat /var/log/dnf_sles_es_migration.log_|-run")
    private StateApplyResult<CmdResult> liberateDnfEs;
 
-   @SerializedName("cmd_|-spmigration_|-cat /var/log/dnf_sll_migration.log_|-run")
+   @SerializedName("cmd_|-spmigration_|-/usr/bin/cat /var/log/dnf_sll_migration.log_|-run")
    private StateApplyResult<CmdResult> liberateDnfSll;
```

Comment:
It seems to me that these are broken also on 5.0 ([Java class](https://github.com/SUSE/spacewalk/blob/2ce3bf3b9511613dc4473eed6d1347e5e7ba6d10/java/code/src/com/suse/manager/webui/utils/salt/custom/DistUpgradeSlsResult.java#L35-L42), [salt state](https://github.com/SUSE/spacewalk/blob/2ce3bf3b9511613dc4473eed6d1347e5e7ba6d10/susemanager-utils/susemanager-sls/salt/distupgrade/init.sls#L26-L30)) and 4.3 ([Java class](https://github.com/SUSE/spacewalk/blob/Manager-4.3/java/code/src/com/suse/manager/webui/utils/salt/custom/DistUpgradeSlsResult.java#L35-L42), [state file](https://github.com/SUSE/spacewalk/blob/Manager-4.3/susemanager-utils/susemanager-sls/salt/distupgrade/init.sls#L26-L30))

Reply:
yes, maybe you can also do a fix for it later.